libs/openFrameworks/utils/ofSystemUtils.h

```diff
@@ -11,13 +11,13 @@ class ofFileDialogResult{
 
 	/// \return the name of the selected file or directory, if set
 	/// currently returns only 1 file, this may change in the future
-	of::filesystem::path getName();
+	std::string getName();
 
 	/// \return the full path of the selected file or directory, if set
-	of::filesystem::path getPath();
+	std::string getPath();
 
-	of::filesystem::path filePath { "" }; ///< full path to selected file or directory
-	of::filesystem::path fileName { "" }; ///< selected file or directory name
+	std::string filePath = ""; ///< full path to selected file or directory
+	std::string fileName = ""; ///< selected file or directory name
 	bool bSuccess = false; ///< true if the dialog action was successful, aka file select not cancel
```

Comment:
I think the above could all be of::filesystem::path internally and then `std::string getName(); / std::string getPath();` just uses the `filePath.string()` in its return.

Reply:
Yes or ofPathToString(filePath)

Reply:
hey @ofTheo just for your information, this was reverted back to string . just unsure if it was the idea

Reply:
Yes - I think the issue was that because the members were public they had to be string for usage not to break on Windows. 

My thought was we could do a new PR and maybe have these std::string variables as legacy support but have a `protected` `of::filesystem::path` internally be used by the class with the string values copies of the path data. 

And then a method for accessing as a file path. 

But it felt like it needed a bit of thought and we needing to revert to string because of the direct variable usage anyway.